core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -418,4 +418,9 @@ export class DotEditContentFormComponent implements OnInit {
 
         window.open(realUrl, '_blank');
     }
+
+    onActiveIndexChange($event: TabViewChangeEvent) {
```

Comment:
@oidacra This `event` is not a signal; therefore, I think the proper naming should be `event` without the `$` symbol avoiding future confusion

Reply:
Changed.